scripts/check_deployment_secrets.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""
+Script to ensure that the secrets in the various HCA stage deployments are not accidentally
+changed to personal user credentials (or otherwise).
+"""
+import subprocess
+import os
+import json
+
+
+auth_uri = ['https://auth.data.humancellatlas.org/authorize', 'https://auth.dev.data.humancellatlas.org/authorize']
+token_uri = ['https://auth.data.humancellatlas.org/oauth/token','https://auth.dev.data.humancellatlas.org/oauth/token']
+dev_email = ['travis-test@human-cell-atlas-travis-test.iam.gserviceaccount.com']
+integration_email = ['org-humancellatlas-integration@human-cell-atlas-travis-test.iam.gserviceaccount.com']
+staging_email = ['org-humancellatlas-staging@human-cell-atlas-travis-test.iam.gserviceaccount.com']
```

Comment:
These values should be grabbed from our Terraform state file, which can be accomplished via output values.

See:
https://www.terraform.io/docs/configuration/outputs.html

Reply:
Should be addressed!  Thanks @xbrianh .